app/src/main/java/io/github/sds100/keymapper/util/MathUtils.kt

```diff
@@ -52,3 +54,21 @@ fun movePointByDistanceAndAngle(p: Point, distance: Int, degrees: Double): Point
 fun angleBetweenPoints(p1: Point, p2: Point): Double {
     return rad2deg(atan2((p2.y - p1.y).toDouble(), (p2.x - p1.x).toDouble()))
 }
+
+fun distributePointsOnCircle(circleCenter: Point, circleRadius: Float, numPoints: Int): ArrayList<Point> {
```

Comment:
Use `List` instead of ArrayList in Kotlin.

Reply:
fixed